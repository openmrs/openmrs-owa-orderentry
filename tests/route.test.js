import React from 'react';
import Routes from '../app/js/routes';

let mountedComponent;

const getComponent = () => {
  if (!mountedComponent) {
    mountedComponent = shallow(<Routes />);
  }
  return mountedComponent;
};

describe('Routes', () => {
  it('successfully rendered', () => {
    expect(getComponent()).toMatchSnapshot();
  });
});
