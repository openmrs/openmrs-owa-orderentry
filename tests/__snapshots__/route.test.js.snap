// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Routes successfully rendered 1`] = `
<div>
  <Connect(Header) />
  <Switch>
    <Route
      component={[Function]}
      path="/"
    />
  </Switch>
</div>
`;
